jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/workitem/ServiceTaskViewImpl.java

```diff
@@ -20,17 +20,20 @@
 import javax.enterprise.context.Dependent;
 import javax.inject.Inject;
 
+import org.gwtbootstrap3.client.ui.Button;
```

Comment:
try using elemental2.dom.HTMLButtonElement instead

Reply:
It's work. Thanks